src/pages/settings/Profile/PersonalDetails/AddressPage.js

```diff
@@ -70,17 +70,25 @@ function updateAddress(values) {
 
 function AddressPage({privatePersonalDetails, route}) {
     usePrivatePersonalDetails();
-    const {translate} = useLocalize();
+    const { translate } = useLocalize();
+    const address = useMemo(() => lodashGet(privatePersonalDetails, 'address') || {}, [privatePersonalDetails]);
     const countryFromUrl = lodashGet(route, 'params.country');
-    const [currentCountry, setCurrentCountry] = useState(countryFromUrl || PersonalDetails.getCountryISO(lodashGet(privatePersonalDetails, 'address.country')));
-    const isUSAForm = currentCountry === CONST.COUNTRY.US;
+    const [currentCountry, setCurrentCountry] = useState(address.country);
     const zipSampleFormat = lodashGet(CONST.COUNTRY_ZIP_REGEX_DATA, [currentCountry, 'samples'], '');
     const zipFormat = translate('common.zipCodeExampleFormat', {zipSampleFormat});
-
-    const address = lodashGet(privatePersonalDetails, 'address') || {};
+    const isUSAForm = currentCountry === CONST.COUNTRY.US;
     const isLoadingPersonalDetails = lodashGet(privatePersonalDetails, 'isLoading', true);
     const [street1, street2] = (address.street || '').split('\n');
     const [state, setState] = useState(address.state);
+    
+    useEffect(() => {
```

Comment:
It's misaligned and there's a space required after if statement (same as throughout the app)
```
useEffect(() => {
    if (!address) {
        return 
    }
    setState(address.state);
    setCurrentCountry(address.country);
}, [address]);
```